cyclops/query/interface.py

```diff
@@ -80,14 +80,16 @@ def run(
 
         return self.data
 
-    def save(self, path: str, file_format: str = "parquet") -> str:
+    def save(
+        self, path: str, file_format: Literal["parquet", "csv"] = "parquet"
```

Comment:
How about some of that pathlib sweetness?

```suggestion
        self, path: Union[str, pathlib.Path], file_format: Literal["parquet", "csv"] = "parquet"
```